lib/ansible/modules/network/eos/eos_static_routes.py

```diff
@@ -0,0 +1,441 @@
+                required: True
+              next_hops:
+                description:
+                  - Details of route to be taken.
+                type: list
+                elements: dict
+                suboptions:
+                  forward_router_address:
+                    description:
+                      - Forwarding router's address on destination interface.
+                    type: str
+                  interface:
+                    description:
+                      - Outgoing interface to take. For anything except 'null0', then next hop IP address should also be configured.
+                      - IP address of the next hop router or
+                      - null0 Null0 interface or
+                      - ethernet e_num Ethernet interface or
+                      - loopback l_num Loopback interface or
+                      - management m_num Management interface or
+                      - port-channel p_num
+                      - vlan v_num
+                      - vxlan vx_num
+                      - Nexthop-Group  Specify nexthop group name
+                      - Tunnel  Tunnel interface
+                      - vtep  Configure VXLAN Tunnel End Points
+                    type: str
+                  nexthop_grp:
+                    description:
+                        - Nexthop group
+                    type: str
+                  admin_distance:
+                    description:
+                      - Preference or administrative distance of route (range 1-255).
+                    type: int
+                  description:
+                    description:
+                      - Name of the static route.
+                    type: str
+                  tag:
+                    description:
+                      - Route tag value (ranges from 0 to 4294967295).
+                    type: int
+                  track:
+                    description:
+                      - Track value (range 1 - 512). Track must already be configured on the device before adding the route.
+                    type: str
+                  mpls_label:
+                    description:
+                      - MPLS label
+                    type: int
+                  vrf:
+                    description:
+                      - VRF of the destination.
+                    type: str
+  running_config:
+    description:
+      - The module, by default, will connect to the remote device and
+        retrieve the current running-config to use as a base for comparing
+        against the contents of source. There are times when it is not
+        desirable to have the task get the current running-config for
+        every task in a playbook.  The I(running_config) argument allows the
+        implementer to pass in the configuration to use as the base
+        config for comparison. This value of this option should be the
+        output received from device by executing command
+    version_added: "2.10"
```

Comment:
`version_added` is not required in this case, as this module itself will be added in 2.10.